app/src/components/market/market_wizard_creator.tsx

```diff
@@ -152,13 +162,10 @@ export class MarketWizardCreator extends Component<Props, State> {
             back={() => this.back()}
             next={() => this.next()}
             values={{
-              outcomeProbabilityOne,
-              outcomeProbabilityTwo,
-              outcomeValueOne,
-              outcomeValueTwo,
+              outcomes: marketData.outcomes,
```

Comment:
PP: Maybe you can destructure outcomes var in the line 138, and use it right here

Reply:
Good catch, fixed.